src/components/Modal/ModalDialog.tsx

```diff
@@ -28,22 +28,20 @@ const ModalDialog = ({ children }: PropsWithChildren) => {
   }, [handleCloseModal])
 
   useEffect(() => {
-    setContainerWidth(divRef.current?.offsetWidth || 0)
+    const offsetWidth = divRef.current?.offsetWidth
+    if (offsetWidth) {
+      containerWidthRef.current = divRef.current.offsetWidth
+    }
     return () => {
-      setContainerWidth(0)
+      containerWidthRef.current = 0
     }
-  }, [divRef.current, width])
+  }, [divRef.current?.offsetWidth])
 
   return (
     <Dialog.Root open={open} onOpenChange={setOpen}>
+      <Dialog.Title></Dialog.Title>
       <Dialog.Content
-        maxWidth={
-          containerWidth
-            ? containerWidth < 768
-              ? "100%"
-              : defaultMaxWidth
-            : defaultMaxWidth
-        }
+        maxWidth={screenWidth < 768 ? "100%" : defaultMaxWidth}
```

Comment:
Is it possible to make it with tailwind somehow or classnames just get ignored by the component (even with !important?) ?

Reply:
No, because it is Radix UI component which expected to get number.